restful_api/tests/test_official_enroll.py

```diff
@@ -0,0 +1,18 @@
+
+from flask import url_for
+
+def test_get_all_enrollments(client, db, official_enroll_factory, test_user_headers):
```

Comment:
If you want can add a `404` no engineer scenerio and engineer found but no course scenerio. But don't have to.